stdlib/socket.pyi

```diff
@@ -390,39 +390,43 @@ class AddressFamily(IntEnum):
     AF_INET: int
     AF_INET6: int
     AF_AAL5: int
-    AF_ALG: int
     AF_APPLETALK: int
     AF_ASH: int
     AF_ATMPVC: int
     AF_ATMSVC: int
     AF_AX25: int
-    AF_BLUETOOTH: int
     AF_BRIDGE: int
-    AF_CAN: int
     AF_DECnet: int
     AF_ECONET: int
     AF_IPX: int
     AF_IRDA: int
     AF_KEY: int
-    AF_LINK: int
     AF_LLC: int
     AF_NETBEUI: int
-    AF_NETLINK: int
     AF_NETROM: int
-    AF_PACKET: int
     AF_PPPOX: int
-    AF_QIPCRTR: int
-    AF_RDS: int
     AF_ROSE: int
     AF_ROUTE: int
     AF_SECURITY: int
     AF_SNA: int
     AF_SYSTEM: int
-    AF_TIPC: int
     AF_UNSPEC: int
-    AF_VSOCK: int
     AF_WANPIPE: int
     AF_X25: int
+    if sys.platform == "linux":
+        AF_CAN: AddressFamily
```

Comment:
Shouldn't these all be annotated with `int`, like the other `AddressFamily` members?

Reply:
Yup it should, thanks for catching!